samples/net/wifi/boards/rd_rw612_bga.conf

```diff
@@ -75,33 +75,6 @@ CONFIG_NET_SOCKETS_SERVICE_THREAD_PRIO=3
 CONFIG_NET_TC_SKIP_FOR_HIGH_PRIO=y
 CONFIG_NET_CONTEXT_PRIORITY=y
 
-CONFIG_WIFI_NM_WPA_SUPPLICANT=y
```

Comment:
but without this will Wi-Fi work on RW612?

Reply:
remove this commit